packages/contracts-bedrock/src/safe/LivenessModule2.sol

```diff
@@ -0,0 +1,323 @@
+    error LivenessModule2_UnauthorizedCaller();
+
+    /// @notice Error for invalid response period.
+    error LivenessModule2_InvalidResponsePeriod();
+
+    /// @notice Error for invalid fallback owner.
+    error LivenessModule2_InvalidFallbackOwner();
+
+    /// @notice Error for when trying to clear configuration while module is enabled.
+    error LivenessModule2_ModuleStillEnabled();
+
+    /// @notice Error for when ownership transfer verification fails.
+    error LivenessModule2_OwnershipTransferFailed();
+
+    /// @notice Emitted when a Safe enables the module.
+    /// @param safe The Safe address that enabled the module.
+    /// @param livenessResponsePeriod The duration in seconds that Safe owners have to
+    ///                               respond to a challenge.
+    /// @param fallbackOwner The address that can initiate challenges and claim ownership if
+    ///                      the Safe is unresponsive.
+    event ModuleEnabled(address indexed safe, uint256 livenessResponsePeriod, address fallbackOwner);
+
+    /// @notice Emitted when a Safe disables the module.
+    /// @param safe The Safe address that disabled the module.
+    event ModuleDisabled(address indexed safe);
+
+    /// @notice Emitted when a challenge is started.
+    /// @param safe The Safe address that started the challenge.
+    /// @param challengeStartTime The timestamp when the challenge started.
+    event ChallengeStarted(address indexed safe, uint256 challengeStartTime);
+
+    /// @notice Emitted when a challenge is cancelled.
+    /// @param safe The Safe address that cancelled the challenge.
+    event ChallengeCancelled(address indexed safe);
+
+    /// @notice Emitted when ownership is transferred to the fallback owner.
+    /// @param safe The Safe address that succeeded the challenge.
+    /// @param fallbackOwner The address that claimed ownership if the Safe is unresponsive.
+    event ChallengeSucceeded(address indexed safe, address fallbackOwner);
+
+    /// @notice Semantic version.
+    /// @custom:semver 2.0.0
+    string public constant version = "2.0.0";
+
+    /// @notice Returns challenge_start_time + liveness_response_period if challenge exists, or
+    ///         0 if not.
+    /// @param _safe The Safe address to query.
+    /// @return The challenge end timestamp, or 0 if no challenge.
+    function getChallengePeriodEnd(address _safe) public view returns (uint256) {
+        uint256 startTime = challengeStartTime[_safe];
+        if (startTime == 0) {
+            return 0;
+        }
+        ModuleConfig storage config = safeConfigs[_safe];
+        return startTime + config.livenessResponsePeriod;
+    }
+
+    /// @notice Configures the module for a Safe that has already enabled it.
+    /// @param _config The configuration parameters for the module containing the response
+    ///                period and fallback owner.
+    function configure(ModuleConfig memory _config) external {
```

Comment:
Likewise, this should be `configureLivenessModule`

Reply:
Fixed in 5be1c2d767b9f6b9ff66edad9497b285d1502d82